src/main/java/org/folio/rest/repository/OwnerRepository.java

```diff
@@ -12,7 +12,7 @@
 
 public class OwnerRepository {
 
-  private static final String OWNERS_TABLE = "owners";
+  public static final String OWNERS_TABLE = "owners";
```

Comment:
please duplicate these constants in test classes

Reply:
done